src/pyfaf/actions/save_reports.py

```diff
@@ -372,3 +375,5 @@ def tweak_cmdline_parser(self, parser):
         parser.add_argument("--speedup", action="store_true",
                             default=False, help="Speedup the processing. "
                             "May be less accurate.")
+        parser.add_argument("--pattern", help="Save reports matched with pattern. "
+                            "Does not work with speedup. E.g. --pattern \"123*\"")
```

Comment:
Why not create a mutual exclusive group then? 

https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group